ksqldb-engine/src/test/java/io/confluent/ksql/function/udf/array/EntriesTest.java

```diff
@@ -123,23 +122,25 @@ public void shouldReturnNullListForNullMapString() {
     assertNull(entriesUdf.entriesString(null, false));
   }
 
-  private <T> void shouldComputeEntries(
-      final Map<String, T> map, final Supplier<List<Struct>> supplier
-  ) {
+  @Test
+  public void shouldReturnNullListForNullMapStruct() {
+    assertNull(entriesUdf.entriesStruct(null, false));
+  }
```

Comment:
Could you add a couple of positive tests `shouldComputeStructEntries` and `shouldComputeStructEntriesSorted`?

Reply:
Added